distribution/docs/src/main/resources/_contents/_processing-framework/processing-framework-intro-contents.adoc

```diff
@@ -27,6 +73,58 @@ asynchronous processing starts and ends, those changes will be overwritten by th
 . `org.codice.ddf.catalog.async.data.api.internal.ProcessResoure`
 . `org.codice.ddf.catalog.async.data.api.internal.ProcessResourceItem`
 
+.Processing Framework Interface Diagram
+[ditaa,processing_framework_interface_diagram,png]
+....
++-----------------------------------------------------------------------------------------------+
+|         +------------------------------------------------------------+                        |
+|         |c369               <PostProcessPlugin>                      |                        |
+|         +------------------------------------------------------------+                        |
+|         |ProcessRequest{ProcessCreateItem} process(ProcessCreateItem)|                        |
+|         |ProcessRequest{ProcessUpdateItem} process(ProcessUpdateItem)|                        |
+|         |ProcessRequest{ProcessDeleteItem} process(ProcessDeleteItem)|                        |
+|         +------------------------------------------------------------+                        |
+|                                                                                               |
+|                               +-------------------------------------------------+             |
+|                               |c369 <ProcessRequest{T extends Processitem}>     |             |
+|                               +-------------------------------------------------+             |
+|                               |List{T} getProcessItems                          |             |
```

Comment:
The whole line is deleted when trying to use `List<T>` inside ditaa, so we settled with `List{T}` for now.